hive.yaml

```diff
@@ -0,0 +1,100 @@
+package:
+  name: hive
+  version: 4.0.0
+  epoch: 0
+  description: Apache Hive
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - openjdk-8-default-jvm
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - maven
+      - openjdk-8
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+    HIVE_VERSION: 4.0.0
+    HADOOP_VERSION: 3.3.6
```

Comment:
Since we need to explicitly set the version here, when we package 3.3 could we make sure we're pinned to this version so that there isn't drift?

I.E.,

```
dependencies:
  runtime:
    - hadoop=3.3.6
```

Same for Tez

Reply:
Totally makes sense, thanks @EyeCantCU.